.travis.yml

```diff
@@ -36,8 +36,6 @@ install:
 - test $MASTER && sed -ie 's#http://downloads.plone.org.br/release/.*/#http://dist.plone.org/release/4.3-latest/#' buildout.d/base.cfg || true
 - test $MASTER && sed -ie '/buildout\.d\/versions\.cfg/d' buildout.cfg || true
```

Comment:
Como [você removeu `buildout.d/versions.cfg` de travis.cfg](https://github.com/plonegovbr/portal.buildout/pull/89/files#diff-a5f387c18d0fec2c7be746751ba26db3L4), esse sed não é mais necessário. Mas ficou uma dúvida, como você agora herda de `development.cfg`, não seria o caso de adaptar esse `sed` para remover `buildout.d/versions.cfg` de `development.cfg` quando for o `MASTER`?

Reply:
eu não removi o `buildout.d/versions.cfg` do Travis; ele continua sendo usado no build de desenvolvimento que é esse chamado de `LAST_RELEASE`.

fica complicado entender, ne? ;)

Reply:
Eu não disse que você removeu do repositório, e sim do arquivo travis.cfg (https://github.com/plonegovbr/portal.buildout/blob/67cea013495833a38368e6883bcaaf451a0d4250/travis.cfg#L13), onde você alterou pra usar o `etc/versions.cfg`. Por isso questionei do sed pra remover essa linha do `development.cfg` quando for testar o `MASTER`.

Reply:
deixei comentários no código para vocês entenderem isso.